ui/apps/platform/src/queries/serviceAccount.js

```diff
@@ -13,6 +13,7 @@ export const SERVICE_ACCOUNT_FRAGMENT = gql`
         }
         clusterName
         clusterId
+        clusterAdmin
```

Comment:
I did not comb through the file history thoroughly, but from the spot-checking I did, I think your guess is correct--this has never worked. 😞

Reply:
Okay, it has been broken since 2019-08-01 in rox 2729 **Config mgmt list refactor**

Which added `if` statement with early return
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/ConfigManagement/List/List.js#L148-L162

Therefore instead of `SERVICE_ACCOUNTS_QUERY` which has `clusterAdmin`
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/queries/serviceAccount.js#L50

And is imported here:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/ConfigManagement/List/ServiceAccounts.js#L13

Is not used, because instead the entity, Role in this specific case, calls `getFragmentInfo`
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/ConfigManagement/Entity/Role.js#L68-L72

Which calls `getFragment` which uses `SERVICE_ACCOUNT_FRAGEMENT` instead
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/utils/queryService.js#L243

Although it might be a lower priority to determine if any of the queries are orphans, it seems worthwhile to compare the plural queries to the fragment queries for all of the entities in case any other fields are missing.